holidays/countries/india.py

```diff
@@ -134,6 +191,50 @@ def _populate_public_holidays(self):
             2028: (OCT, 17),
             2029: (NOV, 5),
             2030: (OCT, 26),
+            2031: (NOV, 14),
+            2032: (NOV, 2),
+            2033: (OCT, 22),
+            2034: (NOV, 10),
+            2035: (OCT, 30),
+        }
+
+        # https://www.timeanddate.com/holidays/india/govardhan-puja
+        govardhan_puja_dates = {
+            2001: (NOV, 15),
+            2002: (NOV, 5),
+            2003: (OCT, 26),
+            2004: (NOV, 13),
```

Comment:
IMO it would be nice if we could move these new Hindu holidays to the dedicated `hindu` calendar and calendar group.

For coding examples, see `sinhala` implementation in #2228